quick-tabs/background.js

```diff
@@ -481,7 +485,7 @@ function initBadgeIcon() {
  */
 function updateBadgeText() {
   if (showTabCount()) {
-    var val = tabs.filter(includeTab).length;
+    var val = tabs.filter(validTab).filter(includeTab).length;
```

Comment:
Might I suggest that you run both functions together rather than using a second array method, as every chained array method means the creation of a new array.

```suggestion
    var val = tabs.filter(tab => validTab(tab) && includeTab(tab)).length;
```

It is a small performance optimization for sure but limiting the use of unnecessary array methods can add up over time with lowering memory usage.

Reply:
Sure thing! Fixed that.